lib/editing/editor_codemirror.css

```diff
@@ -169,7 +169,19 @@ li.CodeMirror-hint-active {
 
 .type-no_suggestions::before {
   content: "?";
-  line-height: 14px; /*for vertical centering */
   background: radial-gradient(#fff, #eee);
-  vertical-align: text-bottom;
 }
+
+/*Chrome specific styling hack*/
```

Comment:
Can you also reference the issue here, for more context when somebody sees this code? Just a #number would work.